### ISSUE TITLE
Missing Legend

### ISSUE THREAD

**Issue by ghost** [cis-bathnes/pattern-lib#17]
![image](https://user-images.githubusercontent.com/609519/34829273-0c591f7c-f6d8-11e7-9531-a003f46f7501.png)
Legend (title) missing. Hint text alone does not describe fields and cannot be used.

Reference: Templates > Question Pages
